src/school/model/School.ts

```diff
@@ -1,4 +1,9 @@
 export class School {
+  updateData(cityId: number, name: string) {
```

Comment:
Puedes mover el metodo. A mi personalmente me gusta asi

clase 
     attributos
     constructores
     metodos
    getters


```suggestion
export class School {

  private _id: number;
  private _cityId: number;
  private _name: string;

  constructor(id: number, cityId: number, name: string) {
    this._id = id;
    this._cityId = cityId;
    this._name = name;
  }

  updateData(cityId: number, name: string) {
    this._cityId = cityId;
    this._name = name;
  }

  get id(): number {
    return this._id;
  }

  get cityId(): number {
    return this._cityId;
  }

  get name(): string {
    return this._name;
  }
}
```